modules/android/pages/vector-search-api-reference.adoc

```diff
@@ -0,0 +1,286 @@
+= Vector Search API Reference
+:page-aliases: 
+ifdef::show_edition[:page-edition: {release}]
+ifdef::prerelease[:page-status: {prerelease}]
+:page-role:
+:description: API reference for Vector Search in Couchbase Lite.
+:keywords: edge AI api swift ios macos apple vector search generative
+
+== Vector Index API
+
+You can use the Vector Index API's core functionality to efficiently perform similarity search at scale on Vector Indexes.
+You can make use of new qurry functions such as `vector_match()` to query indexed vectors.
+
+=== Vector Encoding
+
+Vector encoding is the process of representing data points such as images, words, or documents as vectors in high-dimensional space where each dimension represents a different attribute of the data. 
+Encoding is a crucial step in preparing data for similarity or nearest neighbour search operations.
```

Comment:
```suggestion
Encoding is a crucial step in preparing data for similarity or nearest-neighbor search operations.
```

Watch the British spelling.

Reply:
Not sure if `Vector Encoding` mentioned in this section is about the vector encoding methods used in the vector index or not. If it is, Vector Encoding is the method or algorithm that compresses the vectors stored in the vector index. The encoding makes the index size smaller and reduce the I/P during index and query time, however, on the flip side, encoding could make the distance computation and query result less inaccurate.

We support 3 types of encoding in our API: None (No encoding, No data-loss ), SQ and PQ. Here are a bit of info about the SQ and PQ encoding:

https://github.com/couchbaselabs/mobile-vector-search/blob/main/docs/Tuning.md#2-vector-encoding